sandboxes/staticanalysis/staticanalyze.go

```diff
@@ -9,16 +9,16 @@ import (
 	"time"
 
 	"github.com/ossf/package-analysis/internal/log"
-	"github.com/ossf/package-analysis/internal/pkgecosystem"
+	"github.com/ossf/package-analysis/internal/pkgmanager"
 	"github.com/ossf/package-analysis/internal/staticanalysis"
 	"github.com/ossf/package-analysis/internal/staticanalysis/parsing"
 	"github.com/ossf/package-analysis/internal/utils"
 	"github.com/ossf/package-analysis/internal/worker"
-	"github.com/ossf/package-analysis/pkg/api"
+	"github.com/ossf/package-analysis/pkg/api/pkgecosystem"
 )
 
 var (
-	ecosystem   = flag.String("ecosystem", "", "Package ecosystem (required)")
+	ecosystem   pkgecosystem.Ecosystem
```

Comment:
And after it's made into a function, we can make it look like line 27 (ecosystem = ...)

Reply:
See my other comment